examples/DampedPendulum/main.py

```diff
@@ -0,0 +1,208 @@
+    params.learningRate = learning_rate
+
+    return params
+
+
+def train_SciNet(model, train_O, train_Q, train_A):
+    print("\nTraining Model...")
+
+    n_observations = train_O.shape[0]
+    batch_length = int(np.ceil(batch_size * n_observations))
+    train_inds = np.array(list(range(n_observations)))
+
+    losses = []
+    for epoch in range(num_epochs):
+
+        tmp_train_inds = list(
+            np.random.choice(
+                train_inds,
+                size=(batch_length),
+                replace=False
+            )
+        )
+        tmp_train_O = train_O[tmp_train_inds, :]
+        tmp_train_Q = train_Q[tmp_train_inds, :]
+        tmp_train_A = train_A[tmp_train_inds, :]
+        
+        tmploss = model.train(tmp_train_O, tmp_train_Q, tmp_train_A, batch_length)
+        losses.append(tmploss)
+
+        if (not (epoch) % display_epoch):
+            print(f"EPOCH: {epoch:02d} of {num_epochs}.\tLOSS: {tmploss}")
+
+    return losses
+
+
+def test_SciNet(model, test_O, test_Q, test_A):
+    print("\nTesting Model...")
+
+    losses, activation = model.test(test_O, test_Q,  test_A)
+
+    return losses, activation
+
+
+def main(input_file):
+    # ==============================================================
+    # Load the data from the generated file
+    # ==============================================================
+    spring_consts, damp_consts, O = load_data(input_file)
+
+    train_O, test_O, train_Q, test_Q, train_A, test_A = split_and_format_data(O)
+    n_points = train_O.shape[1]
+
+    # ==============================================================
+    # Create scinet object
+    # ==============================================================
+
+    # initialize parameters
+    params = load_hyperparams(n_points)
+
+    # initialize scinet
+    model = scinet.Scinet(params)
+    model.float()
+
+    # ==============================================================
+    # Train SciNet
+    # ==============================================================
+    losses = train_SciNet(model, train_O, train_Q, train_A)
+
+    # ==============================================================
+    # Test time!
+    # ==============================================================    
+    model_A = model.forward(test_O, test_Q)[-1].detach().numpy().ravel()
+    testy_A = test_A.numpy().ravel()
+
+    fig, ax = plt.subplots()
+
+    ax.set_aspect('equal')
+    ax.plot([-1, 1], [-1, 1], 'k-')
+    ax.plot(testy_A, model_A, 'bo')
+    ax.set_xlabel("Simulated Position")
+    ax.set_ylabel("SciNet Position")
+
+    # plt.show()
+
+    avgLoss, activation = test_SciNet(model, test_O, test_Q, test_A)
+
+    # ==============================================================
+    # Visualization!
+    # ==============================================================  
+      
+    vis = scinet.visualization
+
+    plot_loss = vis.plot_loss(num_epochs, losses) 
+    plot_latent = vis.plot_latent(param1, param2, activation)
```

Comment:
What is param1 and param2? They aren't defined.
I'm trying not to touch the DampedPendulum files much cause that's not really what this PR is about, but this is just a straight up error. If not here, it should be addressed soon

Reply:
I'll leave it for now, I'm really leaving the example stuff up to you guys, I'll trust you to figure it out in the actual `DampedPendulum` branch

Reply:
The params are for the user to put in the input parameters needed to be plotted. For the damped pend, it would be the damp const and spring const. It's something that can be done within the examples themselves :)

Reply:
Yeah, in the visualization functions I get that, but this is in the damped pendulum example, when it's actually being used

Reply:
I'll leave it to you guys to fix in the `DampedPendulum` branch, just wanted to make sure you saw this bug.